fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionSetupTest.java

```diff
@@ -225,34 +123,17 @@ public void timestampsCantBeNegative() {
         IllegalArgumentException.class,
         () ->
             new ElectionSetup(
-                writeIn,
-                electionSetupName,
-                creation,
-                -1,
-                end,
-                votingMethod,
-                laoId,
-                ballotOptions,
-                question,
-                ElectionVersion.OPEN_BALLOT));
+                ELECTION_NAME, CREATION, -1, END, laoId, ElectionVersion.OPEN_BALLOT, questions));
     assertThrows(
         IllegalArgumentException.class,
         () ->
             new ElectionSetup(
-                writeIn,
-                electionSetupName,
-                creation,
-                start,
-                -1,
-                votingMethod,
-                laoId,
-                ballotOptions,
-                question,
-                ElectionVersion.OPEN_BALLOT));
+                ELECTION_NAME, CREATION, START, -1, laoId, ElectionVersion.OPEN_BALLOT, questions));
   }
 
   @Test
   public void electionSetupEqualsTrueForSameInstance() {
+    //noinspection EqualsWithItself
```

Comment:
I don't understand this comment sorry

Reply:
Just Android Studio saying that there is an error here : the equality is made with itself